app/models/venue.rb

```diff
@@ -18,6 +18,16 @@ class Venue < ActiveRecord::Base
   has_many :reviews
   belongs_to :user
 
+  include PgSearch
+  pg_search_scope :search_by_venue_name,
+  against: :venue_name,
+  :using => {
+              :tsearch => {:prefix => true}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Space inside { missing.<br>Space inside } missing.